README.md

```diff
@@ -169,3 +172,17 @@ For `dsub` server documentation, see [servers/dsub](servers/dsub/README.md).
 
 ## Job Manager `cromwell` Server
 For `cromwell` server documentation, see [servers/cromwell](servers/cromwell/README.md).
+
+## Build docker images and releases
+From v0.2.0, Job Manager starts to release stock docker images on [DockerHub](https://hub.docker.com/u/databiosphere)
```

Comment:
I would move this line below the "how to build" instructions, and make it more of a "now upload the results to dockerhub (ask XYZ for access if you don't have it already)"

Reply:
I'd also make "Add a github release pointing to the dockerhub images" a step

Reply:
That sounds better, will do.